tests/test_rounding.py

```
@@ -31,6 +37,24 @@ def round_and_assert(input, expect):
     round_and_assert(Decimal("1.123456789123456789"), Decimal("1.12346"))
 
 
+def test_round_to_requirements_no_min_decimals():
+    def round_and_assert(input, expect):
```

Ruff (0.16.1)

[error] Function argument `input` is shadowing a Python builtin

(A002)